db/schema.rb

```diff
@@ -84,20 +92,20 @@
   create_table "gem_typo_exceptions", force: :cascade do |t|
     t.string "name"
     t.text "info"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
   end
 
   create_table "linksets", id: :serial, force: :cascade do |t|
     t.integer "rubygem_id"
-    t.string "home"
-    t.string "wiki"
-    t.string "docs"
-    t.string "mail"
-    t.string "code"
-    t.string "bugs"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.string "home", limit: 255
+    t.string "wiki", limit: 255
+    t.string "docs", limit: 255
+    t.string "mail", limit: 255
+    t.string "code", limit: 255
+    t.string "bugs", limit: 255
```

Comment:
Rails 7 again maybe? I don't think we want these limits, URLs can technically be arbitrarily long.